beam/__init__.py

```diff
@@ -46,6 +48,43 @@ class SupportedInstallationMethod(Enum):
     CONDA = "conda"
 
 
+def run_beam():
+    argparser = argparse.ArgumentParser()
```

Comment:
Add description that gives links to the default registry

Reply:
```suggestion
    argparser = argparse.ArgumentParser(
        prog="datatractor beam"
        description="""CLI for datatractor extractors that takes a filename and a filetype and runs an appropriate extractor, if available, from the chosen registry (default: https://registry.datatractor.org/).
        
Filetype IDs can be found in the registry API at e.g., https://registry.datatractor.org/api/filetypes. If a matching extractor is found at https://registry.datatractor.org/api/extractors, it will be installed into a virtual environment local to the `beam` installation."""
    )     
```